client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -328,6 +361,94 @@
 
             return api.save('archive_link', {}, data, item);
         };
+
+        /**
+        * Actions that it can perform on an item
+        * @param {Object} item : item
+        * @param {Object} privileges : user privileges
```

Comment:
Oh, just noticed - `privileges` is not a parameter here, it's a service dependency.